README.md

```diff
@@ -1,17 +1,17 @@
 # ORB-SLAM3-Datasets-Preprocessing
 * At first you need to preprocess the dataset. Preprocessing includes the transformation of depth images into RGB images coordinates and converting pictures to undistorted format. You can do this by executing:
 ```
-python preproccesing.py DEPTH_IMAGES_FOLDER COLOR_IMAGES_FOLDER INI_CONFIG_FILE
+python preprocessing.py DEPTH_MASTER_IMAGES COLOR_MASTER_IMAGES DEPTH_SLAVE_IMAGES COLOR_SLAVE_IMAGES INI_CONFIG_FILE
```

Comment:
Looks like on this step there is no need to combine processing of data from all cameras in one call, better to remain call for one pair of color and depth and just run it for every pair independently. As argument you can pass name of the folder where to put things not to have them hardcoded in the code.

Common processing is more native on association step.

Reply:
Also, it looks like we can use only one argument for path to folder where `color` and `depth` are kept, not two as it is now. We can assume that internal folder naming for color and depth will be the constant.